fs.go

```diff
@@ -23,13 +24,13 @@ type FileSystem interface {
 	Reload(func(key, name string)) error
 
 	// Create should return a new File using a name generated from the passed key.
-	Create(key string) (File, error)
+	Create(key string) (stream.File, error)
```

Comment:
Let's just embed the stream.Filesystem in this interface. It'll make it clearer that we need all of the features of stream.